src/main/java/com/exactpro/th2/act/actions/FindMessageInGui.java

```diff
@@ -111,10 +114,12 @@ protected void collectActions(RptViewerSearchDetails rptViewerDetails, TestUIFra
 
 		//waits that event is loaded
 		//expand subroot event
-		builderManager.click().locator(WebLocator.byXPath(ExtractMessage.EVENT_EXPAND_XPATH)).wait(5).build();
+		builderManager.waitForElement().locator(WebLocator.byXPath(EVENT_EXPAND_XPATH)).seconds(5).build();
+		builderManager.waitAction().seconds(1).build();
+		builderManager.click().locator(WebLocator.byXPath(EVENT_EXPAND_XPATH)).build();
```

Comment:
These lines duplicate in ExtractMessage.java. Is it expected and shouldn't be wrapped into a method?